tools/rosgraph/test/test_roslogging.py

```diff
@@ -92,6 +93,7 @@ def test_rosconsole__logging_format():
         base, ext = os.path.splitext(this_file)
         if ext == '.pyc':
             this_file = base + '.py'
+        this_file = re.escape(this_file)
```

Comment:
Please apply the escaping where it is being used - in this case directly in line 114.